gizmo.py

```diff
@@ -0,0 +1,10 @@
+# Test gizmo file
+def hello(name, country='Finland'):
+  print('Hello {name}, how are things in {country}?')
```

Comment:
you forgot the `f` at the beginning of the string

Reply:
Ah yes, didn't see that. Fixed now 👍🏽